HomeWorkShoppingCart/HomeWorkShoppingCart/Repository/CartRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using HomeWorkShoppingCart.Entities;
+
+namespace HomeWorkShoppingCart.Repository
+{
+    public class CartRepository
+    {
+        public CartEntity cartStorage { get; } = new CartEntity();
+
+        public void AddProductCart(string productId, decimal sum)
+        {
+            cartStorage.productIds.Add(productId);
+            cartStorage.sum += sum;
+        }
+
+        public CartEntity GetCart()
+        {
+            return cartStorage;
+        }
+
+        public void ClearCart()
+        {
+            cartStorage.productIds.Clear();
+            cartStorage.sum = 0;
+        }
```

Comment:
Actually we don't need repository for cart as we need this indo to be saved only in RAM